_posts/2021-09-20-newpipe-0.21.10-released.md

```diff
@@ -0,0 +1,112 @@
+
+- A cool new setting was added by [@Stypox](https://github.com/Stypox): You can now automatically have videos playing full screen if you use the main player.
+
+- [@Stypox](https://github.com/Stypox) also split search suggestions into two categories: local and remote. Now you have the option to enable or disable them independently of each other.
+
+- [@magicbrothers](https://github.com/magicbrothers) added a new Invidious instance, Tubus, meant for educational videos for children.
+
+- [@ktprograms](https://github.com/ktprograms) finished the ongoing work of adding the `Show channel details` option where it was still missing, that is, in local playlists, feed items, and history items.
+
+### Improved
+
+- [@sauravrao637](https://github.com/sauravrao637) modified the app colours to be a bit darker when the dark theme is selected in the app, which looks loads better to us! He further added a darker splash screen variant when the OS's theme is set to dark. We hope you like the new look too!
+
+- [@Stypox](https://github.com/Stypox) replaced the outdated Android-Universal-Image-Loader library (which has not had any new release in the last 6 years) with the more stable and updated Picasso, a library to download and cache images on Android (think all the thumbnails you see in the app). This fixes a nasty issue where the wrong notification thumbnail size could crash the whole System UI or even forcibly reboot the device! It also makes loading cached images instantaneous, and loading non-cached images smoother.
+
+- [@sherlockbeard](https://github.com/sherlockbeard) changed the 'video not available' image from a static PNG to a vector XML image, allowing it to scale properly for screens of all sizes.
+
+- [@Stypox](https://github.com/Stypox) added the mime-type field to all calls to the SAF file picker, which causes files with the incorrect file format to be greyed out and become non-selectable. This will ensure the user can only choose a ZIP file when importing their database, for example. This should be a great quality-of-life improvement for many users!
+
+### Fixed
+
+- [@litetex](https://github.com/litetex) together with [@Stypox](https://github.com/Stypox) improved the comments parsing for timestamps, which could have resulted in a crash if it happened in the wrong place at the wrong time
+
+- [@Aga-C](https://github.com/Aga-C) added line wrapping to the settings texts, so they don't get truncated anymore.
+
+- [@acti0](https://github.com/acti0) fixed a regression from a past release where the name of the video wasn't being shared when sharing the video URL.
+
+- [@litetex](https://github.com/litetex) fixed a bug where you had to tap the Replay button twice for it to actually work.
+
+- While [@litetex](https://github.com/litetex) fixed the Replay button, they also fixed the direction buttons on DPads for Android TV users: the seekbar now jumps only 10 seconds, instead of several minutes.
```

Comment:
```suggestion
- While [@litetex](https://github.com/litetex) fixed the Replay button, they also fixed the direction buttons on D-Pads for Android TV users: the seekbar now jumps only 10 seconds, instead of several minutes.
```